lasagne/layers/transform.py

```diff
@@ -0,0 +1,198 @@
+import theano.tensor as T
+
+from .base import MergeLayer
+
+__all__ = [
+    "TransformerLayer",
+]
+
+
+class TransformerLayer(MergeLayer):
+    """
+    Spatial transformer layer [1]_.
```

Comment:
The reference shouldn't be part of the first sentence, otherwise you get "reference not found" errors in Sphinx when it's compiling the overview tables for http://lasagne.readthedocs.org/en/latest/modules/layers.html.

It'd actually be good to have a second paragraph of two or three sentences describing what kind of spatial transform this actually performs (i.e., what do the six values mean). That's where you could put the reference.